.php-cs-fixer.dist.php

```diff
@@ -9,6 +9,7 @@
     ])
     ->name('*.php')
     ->notName('*.cache.php')
+    ->notPath('Tempest/Router/src/Route.php') // phpcs doesn't yet support property hooks in interfaces
```

Comment:
Simply commenting to say let's keep an eye on [this PR](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8312).